infra/variables.tf

```diff
@@ -0,0 +1,71 @@
+variable project {
+  default = "Develop-The-Profession-MOCK"
+}
+
+variable environment {
+  default = "dev"
+}
+
+variable aws_region {
+  default = "eu-west-1"
+}
+
+variable aws_account_id {
+  default = "054631451206"
+}
+
+variable mot_DtP_mock_api_enabled {
+  default = true
+}
+
+variable dtp_api_version {
+  default = "v1"
+}
+
+variable logs_retention {
+  default = "30"
+}
+
+variable ABC_usage_plan {
+  default {
+    organization_name     = "ABC"
+    quota_limit           = 1000
+    quota_period          = "DAY"
+    throttle_burst_limit  = 5
+    throttle_rate_limit   = 5
+    api_key_value         = "#CY&#n5fjuI&2CJW7zDChr6R24%Jta"
+  }
+}
+
+variable IMI_usage_plan {
+  default {
+    organization_name     = "IMI"
+    quota_limit           = 1000
+    quota_period          = "DAY"
+    throttle_burst_limit  = 5
+    throttle_rate_limit   = 5
+    api_key_value         = "OA!tTC3@NQTGzx^2e4o#G8d&8OBwei"
```

Comment:
not sure we should put those in here

Reply:
they are defaults used during dev. they will be overwritten during jenkins build.